ccutil/params.cpp

```diff
@@ -102,7 +102,7 @@ bool ParamUtils::SetParam(const char *name, const char* value,
   IntParam *ip = FindParam<IntParam>(name, GlobalParams()->int_params,
                                      member_params->int_params);
   if (ip && ip->constraint_ok(constraint) &&
-      sscanf(value, INT32FORMAT, &intval) == 1) ip->set_value(intval);
+      sscanf(value, "%d", &intval) == 1) ip->set_value(intval);
```

Comment:
There are more places where INT32FORMAT is being used
https://github.com/tesseract-ocr/tesseract/search?q=INT32FORMAT

INT32FORMAT is not being used.